scripts/gen_vimdoc.py

```diff
@@ -449,11 +449,13 @@ def get_text(n, preformatted=False):
         for node in n.childNodes:
             text += get_text(node)
         return '`{}`'.format(text)
+    if n.nodeName == 'sp':
```

Comment:
micro-comment explaining what this is?
```suggestion
    if n.nodeName == 'sp':  # foo
```

Reply:
04eea57b9546f9919c62124a8de8cf4cf788e421